config/initializers/active_job_logger_patch.rb

```diff
@@ -5,6 +5,13 @@
   module ActiveJob
     module Logging
       class LogSubscriber
+        def enqueue(event)
+          info do
+            job = event.payload[:job]
+            "Enqueued #{job.class.name} (Job ID: #{job.job_id}) to #{queue_name(event)} at: #{Time.zone.now}"
```

Comment:
Ah, the other ones are in the worker log. I suppose we should add timestamps there too. What do you think of this format?
```
[ActiveJob] [2017-07-12 22:25:32 UTC] Enqueued SmsOtpSenderJob (Job ID: 502339a0-29da-41fb-b5dc-5c44e1b5df75) to Sidekiq(sms)
```